duckduckgo/macos-browser
# Do not reload DBP tab when switching to it

Task/Issue URL: https://app.asana.com/0/1203581873609357/1206115814306009/f

**Description**:
Do not reload the DBP tab when switching to it

**Steps to test this PR**:
[ Prepare to use DBP]
1. Return `false` from this line https://github.com/duckduckgo/macos-browser/blob/2bd8ca8663b388d62126822c8c26be7e39fdd547/LocalPackages/DataBrokerProtection/Sources/DataBrokerProtection/Services/RedeemCodeServices.swift#L72
2. Return `true` from this line if you're not already logged as in internal user https://github.com/duckduckgo/macos-browser/blob/2bd8ca8663b388d62126822c8c26be7e39fdd547/DuckDuckGo/DBP/DataBrokerProtectionFeatureVisibility.swift#L62
3. Go to the Browser Menu, select Personal information removal
4. Change tabs, get back to it, see if it forces a reload, it shouldn't
5. Close the PII tab, check if `DataBrokerProtectionViewController` was removed from memory, open it again, check if the UI loads again



---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
